### DISCUSSION TITLE
a/using-pre-commit

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#19] [Announcements]
# a/using-pre-commit

https://blog.yasking.org/a/using-pre-commit.html

**Comment by zishiguo** [comment]
大佬，写得不错！请问，能出个commit message 规范校验的文章嘛？

**Reply by sincerefly** [reply]
感谢回复  ❤️  对 commit message 进行校验听起来很棒！我近期比较忙，等发博客不知道什么时候了，建议先自行研究看看

**Comment by zishiguo** [comment]
配置了commitlint-pre-commit-hook， 但是没有安装commit-msg hook，执行如下命令
`pre-commit install --hook-type commit-msg`
https://github.com/alessandrojcm/commitlint-pre-commit-hook
